### PR TITLE
refactor

### PR DESCRIPTION
**Pull request by jetersen**
minor refactoring.

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: app/commands.js [hunk 1]
```diff
@@ -6,18 +6,13 @@ import { RemoveLabelCommand } from "./commands/remove-label-command.js";
...
     new RemoveLabelCommand(id, payload),
     new ReviewerCommand(id, payload),
   ];
-}
-
-export function noneMatch(commands) {
-  return !commands.some((command) => {
-    return command.matches() !== null;
-  });
+  return commands.filter((command) => command.matches());
```
Comment (jetersen) [inline app/commands.js:17]: Thought we could inline the filter right here.

File: app/router.js [hunk 2]
```diff
@@ -6,35 +6,33 @@ import { Octokit } from "@octokit/core";
 import { config as octoKitConfig } from "@probot/octokit-plugin-config";
 
 import deepmerge from "deepmerge";
-import { getCommands, noneMatch } from "./commands.js";
+import { getCommands } from "./commands.js";
 
-export async function router(auth, id, payload, verbose) {
-  const sourceRepo = payload.repository.name;
+const OctokitConfig = Octokit.plugin(octoKitConfig);
```
Comment (jetersen) [inline app/router.js:11]: We only need to declare this class constructor once.

File: app/router.js [hunk 3]
```diff
@@ -6,35 +6,33 @@ import { Octokit } from "@octokit/core";
...
   const authToken = await getAuthToken(auth, payload.installation.id);
-  const OctokitConfig = Octokit.plugin(octoKitConfig);
   const octokit = new OctokitConfig({ auth: authToken });
 
   // TODO validate against schema
   // noinspection JSUnusedGlobalSymbols
   const { config } = await octokit.config.get({
     owner: payload.repository.owner.login,
-    repo: sourceRepo,
+    repo: payload.repository.name,
```
Comment (jetersen) [inline app/router.js:30]: might as well inline this.

File: app/router.js [hunk 4]
```diff
@@ -6,35 +6,33 @@ import { Octokit } from "@octokit/core";
...
     path: ".github/comment-ops.yml",
     defaults: (configs) => deepmerge.all([defaultConfig, ...configs]),
   });
 
-  const runCommands = commands.filter((command) => command.matches());
-  for (const command of runCommands) {
+  for (const command of commands) {
```
Comment (jetersen) [inline app/router.js:35]: the commands are already filtered at this point

File: app/commands.js (hunk 1 above)
Comment (jetersen) [reply app/commands.js:17]: Can  also write it like this:

```javascript
  return [
    new TransferCommand(id, payload),
    new CloseCommand(id, payload),
    new ReopenCommand(id, payload),
    new LabelCommand(id, payload),
    new RemoveLabelCommand(id, payload),
    new ReviewerCommand(id, payload),
  ].filter((command) => command.matches());
```

**Review by timja — APPROVED** [review]
Nice, thanks!

**Pull request merged**